.github/workflows/check-users-in-space-developer-role.yml

```diff
@@ -0,0 +1,70 @@
+on:
+  workflow_dispatch:
+    inputs:
+      CF_SPACE:
+        description: PaaS space to target
+        default: 'bat-prod'
+        required: true
+jobs:
+
+  CF:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: DFE-Digital/github-actions/setup-cf-cli@master
+        with:
+          CF_USERNAME: ${{ secrets.CF_USERNAME }}
+          CF_PASSWORD: ${{ secrets.CF_PASSWORD }}
+          CF_SPACE_NAME: bat-qa
+          # Optional inputs
+          CF_CLI_VERSION: v7
+          CF_ORG_NAME: dfe
+          CF_API_URL:  https://api.london.cloud.service.gov.uk
```

Comment:
can be removed

Reply:
re line Ln 20 and 21; i think we need these because of cf login step (
`cf target -o ${{ inputs.CF_ORG_NAME }} -s ${{ inputs.CF_SPACE_NAME }}`
)
I think we need Ln 19 too, cause it downloads (wget) CF cli

Reply:
the default version if not specified is v7